test/_fixtures/gradle/maven/custom_maven_repo/maven2/mysql/mysql-connector-java/6.0.5/mysql-connector-java-6.0.5.pom

```diff
@@ -0,0 +1,33 @@
+<project>
```

Comment:
are this files too much? I'm using them to test the structure, xml parsing, etc...